<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewGraphDialog</class>
 <widget class="QDialog" name="NewGraphDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>406</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="titleWidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="icons.qrc">:/icons/graph_add.png</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="title_2">
        <property name="styleSheet">
         <string notr="true">QLabel { font-size: 14px; }</string>
        </property>
        <property name="text">
         <string>New Graph</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>159</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="dialogMainWidget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="graphDetailsGroup">
        <property name="title">
         <string>New Graph Details</string>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
         </property>
         <item row="1" column="0">
          <widget class="QLabel" name="graphNameLabel">
           <property name="text">
            <string>Name:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="graphNameEdit"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="graphDirLabel">
           <property name="text">
            <string>Directory:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <layout class="QHBoxLayout" name="graphDirLayout">
           <item>
            <widget class="QLineEdit" name="graphDirEdit"/>
           </item>
           <item>
            <widget class="QPushButton" name="graphDirButton">
             <property name="text">
              <string>Select Directory</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="graphPathLabel">
           <property name="text">
            <string>Path:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLabel" name="graphPath">
           <property name="font">
            <font>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>105</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="buttons" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="standardButtons">
         <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NewGraphDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>463</x>
     <y>385</y>
    </hint>
    <hint type="destinationlabel">
     <x>466</x>
     <y>451</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NewGraphDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>381</x>
     <y>386</y>
    </hint>
    <hint type="destinationlabel">
     <x>389</x>
     <y>449</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>graphNameEdit</sender>
   <signal>textEdited(QString)</signal>
   <receiver>NewGraphDialog</receiver>
   <slot>nameChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>305</x>
     <y>141</y>
    </hint>
    <hint type="destinationlabel">
     <x>552</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>graphDirEdit</sender>
   <signal>textEdited(QString)</signal>
   <receiver>NewGraphDialog</receiver>
   <slot>dirChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>305</x>
     <y>188</y>
    </hint>
    <hint type="destinationlabel">
     <x>545</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>graphDirButton</sender>
   <signal>pressed()</signal>
   <receiver>NewGraphDialog</receiver>
   <slot>selectDir()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>409</x>
     <y>187</y>
    </hint>
    <hint type="destinationlabel">
     <x>532</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>typeChanged(QString)</slot>
  <slot>nameChanged(QString)</slot>
  <slot>dirChanged(QString)</slot>
  <slot>selectDir()</slot>
 </slots>
</ui>
